MasterSeed32.md

```diff
@@ -274,9 +309,38 @@ Generation of long shares and recovery of the master seed from long shares proce
 The long checksum is designed to be an error correcting code that can correct up to 4 character substitutions, up to 8 unreadable characters (called erasures), or up to 15 consecutive erasures.
 As with regular checksums we do not specify how an implementation should implement error correction, and all our recommendations for error correction of regular MS32 strings also apply to long MS32 strings.
 
-## Test Vectors
+# Rationale
+
+Storing master seeds is the most important component of self-custody.
+Users who do not wish to use multisignature setups or sweep their coins must do so by encoding the master secret and physically encoding it.
+The current best practices for this are either to:
+
+* Use BIP39, which suffers from a weak checksum, uses English words which are have inconsistent lengths and near collisions (or other word lists, the choice of which affects the derived secret but is not explicitly encoded anywhere), and has no natural way to add Shamir Secret Sharing; or
+* Use SLIP39, which also uses English words, includes features such as passphrase-based hardening that may not be worth the additional implementation complexity, and does not have a BIP number.
```

Comment:
SLIP-39 (should be hyphenated I think) also has a two level sharing heirarchy that we do not support (in this BIP).

suggest "...includes features such as passphrase-hardening and two level sharing that may not..."